packages/@aws-cdk/aws-apigatewayv2/test/http/stage.test.ts

```diff
@@ -116,4 +116,72 @@ describe('HttpStage', () => {
     const metricNames = metrics.map(m => m.metricName);
     expect(metricNames).toEqual(['4xx', '5xx', 'DataProcessed', 'Latency', 'IntegrationLatency', 'Count']);
   });
+});
+
+describe('HttpStage with domain mapping', () => {
+  const domainName = 'example.com';
+  const certArn = 'arn:aws:acm:us-east-1:111111111111:certificate';
+
+  test('domainUrl returns the correct path', () => {
+    const stack = new Stack();
+    const api = new HttpApi(stack, 'Api', {
+      createDefaultStage: false,
+    });
+
+    const dn = new DomainName(stack, 'DN', {
+      domainName,
+      certificate: Certificate.fromCertificateArn(stack, 'cert', certArn),
+    });
+
+    const stage = new HttpStage(stack, 'DefaultStage', {
+      httpApi: api,
+      domainMapping: {
+        domainName: dn,
+      },
+    });
+
+    expect(stage.domainUrl.endsWith(`${domainName}/`)).toBe(true);
```

Comment:
Should this be an exact match instead? Something like

```ts
    expect(stage.domainUrl).toEqual(`https://${domainName}/`);
```

Reply:
Updated